.env.quicktemplate

```
@@ -0,0 +1,66 @@
+##ORCID Variables for user to set
+OAUTH_CLIENT_ID=<oauth app client id>
+OAUTH_CLIENT_SECRET=<oauth_client_secret goes here>
+
+##Variables already set
+JWT_SECRET=insecure2520f59b2bccc64ce155482bc413f6b9ede5c6d418ab529d1b2051baa6f0849e
+JWT_DOMAIN=localhost
+
+# Database variables
+DB_PASSWORD=postgres
+DB_SERVER=dbserver:5432
+DB_NAME=bidb
+DB_USER=postgres
+
+# Micronaut variables
+API_INTERNAL_PORT=8081
+API_INTERNAL_TEST_PORT=8082
+
+BRAPI_READ_TIMEOUT=60m
+
+# Max number of records to POST to the BrAPI service per request
+POST_CHUNK_SIZE=1000
+
+# Request cache records paginating through available records per program by CACHE_BRAPI_FETCH_PAGE_SIZE
+CACHE_PAGINATE_GERMPLASM=false
+CACHE_BRAPI_FETCH_PAGE_SIZE=65000
+
+# BrAPI Server Variables
+BRAPI_SERVER_PORT=8080
+
+BRAPI_DEFAULT_URL=http://brapiserver:8080
+BRAPI_REFERENCE_SOURCE=breedinginsight.org
+
+BRAPI_DB_SERVER=dbserver:5432
+BRAPI_DB=postgres
+BRAPI_DB_USER=postgres
+BRAPI_DB_PASSWORD=postgres
+
+WEB_BASE_URL=http://localhost
+API_BASE_URL=http://localhost
+
+# Public domain name
+REGISTERED_DOMAIN=localhost
+
+# Email relay information
+EMAIL_RELAY_HOST=mailhog
+EMAIL_RELAY_PORT=1025
+EMAIL_FROM=bidevteam@cornell.edu
+
+REDIS_URL=redis://localhost:6379
```

```suggestion
REDIS_URL=redis://redis:6379
```
Since everything is running in separate docker containers.

---

implemented